saleor/payment/gateways/razorpay/__init__.py

```diff
@@ -88,11 +87,11 @@ def get_client_token(**_):
     return str(uuid.uuid4())
 
 
-def charge(
-        payment: Payment,
-        payment_token: str,
-        amount: Decimal,
-        **connection_params: Dict) -> Tuple[Transaction, str]:
+def process_payment(payment_information: Dict, **connection_params) -> Dict:
```

Comment:
I would put suggest to put `process_payment(...)` under `charge(...)` for better readability. The user that will read the code, won't know yet what is `charge(...)`, which forces them to look around for it.